optuna/testing/visualization.py

```diff
@@ -10,7 +10,13 @@ def prepare_study_with_trials(
     value_for_first_trial: float = 0.0,
 ) -> Study:
 
-    """Prepare a study for tests.
+    """Return a dummy study object for tests.
+
+    This function is added to reduce the code to set up dummy study object in each test case.
+    However, you can only use this function for unit tests that are loosely coupled with the
+    dummy study object. Unittests that are tightly coupled with the study become difficult to
```

Comment:
A nit. Please ignore if `unittest` is intended.
```suggestion
    dummy study object. Unit tests that are tightly coupled with the study become difficult to
```

Reply:
Thank you for your suggestion1